logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -53,6 +60,28 @@ actual class CallManager(
         deferredHandle = startHandleAsync()
     }
 
+    private fun updateCallStatusById(conversationId: String, status: String) {
```

Comment:
Indeed! Our main focus now is making it work for Android and later on when implementing for other targets we can properly look at all the logic we have and what can and can not be extracted to the common package instead of being duplicated for all targets as you mentioned